### PR TITLE
Update 4_inventory_mgmt.md

### PR DESCRIPTION
**Pull request by AditModi**
*Issue #, if available:*

*Description of changes:*

Incorrect documentation, This documentation: https://www.wellarchitectedlabs.com/operational-excellence/100_labs/100_inventory_patch_management/4_inventory_mgmt/

Is out of date.

More details can be found in this issue #452 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: content/Operational Excellence/100_Labs/100_Inventory_Patch_Management/4_inventory_mgmt.md [hunk 1]
```diff
@@ -42,9 +42,11 @@ SSM Agent is installed by default on:
...
-   1. In the **Choose the service that will use this role** section, scroll past the first reference to EC2 (**EC2 Allows EC2 instances to call AWS services on your behalf**) and choose **EC2** from within the field of services. This will open the **Select your use case** section further down the page.
-   1. In the **Select your use case** section, choose **EC2 Role for AWS Systems Manager** to select it.
-   1. Then choose **Next: Permissions**.
+   1. Select EC2 from the common cases option via its radio button, then next.
+   1. Search for the policy "AmazonSSMManagedInstanceCore". Select it and click next.
+   1. Assign role name, add tags if desired, and click create role.
+   1. Please note, if you now view this role from the roles console, and examine the summary, you should now see an instance profile ARN listed on the far right of the       role summary window. This will confirm that you should now be able to select the role from the "Modify IAM role" console.
+   1. return to the instance listing, enter the "Modify IAM role" console: Instances > tick instance in list > Actions > Security > Modify IAM role, and select the           desired role from the drop menu.
```
Comment (awsdunc) [inline content/Operational Excellence/100_Labs/100_Inventory_Patch_Management/4_inventory_mgmt.md:49]: Thanks for the contribution and update. Please can you standardise the language and capitalisation to match the other instruction format 🙏

Comment (AditModi) [reply content/Operational Excellence/100_Labs/100_Inventory_Patch_Management/4_inventory_mgmt.md]: I have made the required changes, please have a look at it.

File: content/Operational Excellence/100_Labs/100_Inventory_Patch_Management/4_inventory_mgmt.md [hunk 2]
```diff
@@ -42,9 +42,8 @@ SSM Agent is installed by default on:
    1. In the navigation pane, choose **Roles**.
    1. Then choose **Create role**.
    1. In the **Select type of trusted entity** section, verify that the default **AWS service** is selected.
-   1. In the **Choose the service that will use this role** section, scroll past the first reference to EC2 (**EC2 Allows EC2 instances to call AWS services on your behalf**) and choose **EC2** from within the field of services. This will open the **Select your use case** section further down the page.
-   1. In the **Select your use case** section, choose **EC2 Role for AWS Systems Manager** to select it.
-   1. Then choose **Next: Permissions**.
+   1. In the Common cases section, **Select EC2** via its radio button.
```
Comment (awsdunc) [inline content/Operational Excellence/100_Labs/100_Inventory_Patch_Management/4_inventory_mgmt.md:45]: "Common use cases"

File: content/Operational Excellence/100_Labs/100_Inventory_Patch_Management/4_inventory_mgmt.md [hunk 3]
```diff
@@ -42,9 +42,8 @@ SSM Agent is installed by default on:
...
+   1. In the next section, **Select the policy "AmazonSSMManagedInstanceCore"**.
 1. Under **Attached permissions policy**, verify that **AmazonEC2RoleforSSM** is listed, and then choose **Next: Tags**.
```
Comment (awsdunc) [inline content/Operational Excellence/100_Labs/100_Inventory_Patch_Management/4_inventory_mgmt.md:47]: This policy is now incorrect?

File: content/Operational Excellence/100_Labs/100_Inventory_Patch_Management/4_inventory_mgmt.md [hunk 4]
```diff
@@ -42,9 +42,8 @@ SSM Agent is installed by default on:
    1. In the navigation pane, choose **Roles**.
    1. Then choose **Create role**.
    1. In the **Select type of trusted entity** section, verify that the default **AWS service** is selected.
-   1. In the **Choose the service that will use this role** section, scroll past the first reference to EC2 (**EC2 Allows EC2 instances to call AWS services on your behalf**) and choose **EC2** from within the field of services. This will open the **Select your use case** section further down the page.
-   1. In the **Select your use case** section, choose **EC2 Role for AWS Systems Manager** to select it.
-   1. Then choose **Next: Permissions**.
+   1. In the Common cases section, **Select EC2** via its radio button.
+   1. In the next section, **Select the policy "AmazonSSMManagedInstanceCore"**.
```
Comment (awsdunc) [inline content/Operational Excellence/100_Labs/100_Inventory_Patch_Management/4_inventory_mgmt.md:46]: Please remove the " " around the policy name and only bold the policy name not "Select the policy"

File: content/Operational Excellence/100_Labs/100_Inventory_Patch_Management/4_inventory_mgmt.md [hunk 5]
```diff
@@ -42,9 +42,8 @@ SSM Agent is installed by default on:
...
 1. Under **Attached permissions policy**, verify that **AmazonEC2RoleforSSM** is listed, and then choose **Next: Tags**.
 1. In the **Tags** section:
```
Comment (awsdunc) [inline content/Operational Excellence/100_Labs/100_Inventory_Patch_Management/4_inventory_mgmt.md:48]: This should probably go after the section where the role is named?

File: content/Operational Excellence/100_Labs/100_Inventory_Patch_Management/4_inventory_mgmt.md (hunk 1 above)
Comment (awsdunc) [reply content/Operational Excellence/100_Labs/100_Inventory_Patch_Management/4_inventory_mgmt.md]: Hey thanks for updating but I think the rest of the steps on this labs need updating to match the new role creation layout if you can load up your changes in hugo, and then run through creating a role and try get them to match please?

**Comment by awsdunc** [comment]
Hey Adit, just checking if you're still working on this PR?

**Comment by awsdunc** [comment]
Closing due to inactivity

**Pull request closed without merging**